pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/CreateSubscriptionTest.java

```diff
@@ -275,10 +276,7 @@ public void addSubscriptionPropertiesTest() throws Exception {
         subscription4 = (PersistentSubscription) pulsar.getBrokerService()
                 .getTopicReference(topic).get().getSubscription(subName);
         properties4 = subscription4.getSubscriptionProperties();
-        assertTrue(properties4.containsKey("3"));
-        assertTrue(properties4.containsKey("4"));
-        assertEquals(properties4.get("3"), "3");
-        assertEquals(properties4.get("4"), "4");
+        assertTrue(properties4.isEmpty());
```

Comment:
`//consumer subscribe without subscriptionProperties set, it will get the old one`
This PR changed this feature. It's worth discussing.

Reply:
ok,I will change it